packages/runtime/runtime-definitions/api-report/runtime-definitions.api.md

```diff
@@ -417,14 +418,16 @@ export interface ISummaryTreeWithStats {
 
 // @public
 export interface ITelemetryContext {
-    // @deprecated
-    get(prefix: string, property: string): TelemetryBaseEventPropertyType;
-    // @deprecated
-    serialize(): string;
```

Comment:
Let's add a changeset announcing the removal of these properties since it's a breaking change.

Reply:
On it